pilot/pkg/features/pilot.go

```diff
@@ -410,6 +410,12 @@ var (
 		"If enabled, Envoy will be configured to prevent traffic directly the the inbound/outbound "+
 			"ports (15001/15006). This prevents traffic loops. This option will be removed, and considered always enabled, in 1.9.").Get()
 
+	EnableDestinationRuleInheritance = env.RegisterBoolVar(
+		"PILOT_DESTINATION_RULE_INHERITANCE",
```

Comment:
nit: PILOT_DESTINATION_RULE_INHERITANCE -> PILOT_ENABLE_DESTINATION_RULE_INHHERITANCE to match with existing env vars

Reply:
done